libs/blocks/caas/utils.js

```diff
@@ -587,15 +587,24 @@ export const getConfig = async (originalState, strs = {}) => {
       },
       button: { style: state.collectionBtnStyle },
       resultsPerPage: state.resultsPerPage,
-      endpoint: `https://${
-        state.endpoint
-      }${targetActivity}?originSelection=${originSelection}&contentTypeTags=${state.contentTypeTags.join(
-        ',',
-      )}&secondSource=${state.showSecondarySource ? state.secondarySource.join(',') : []}&secondaryTags=${state.showSecondarySource ? state.secondaryTags.join(
-        ',',
-      ) : []}&collectionTags=${collectionTags}&excludeContentWithTags=${excludeContentWithTags}&language=${language}&country=${country}&complexQuery=${complexQuery}&excludeIds=${excludedCards}&currentEntityId=&featuredCards=${featuredCards}&environment=&draft=${
-        state.draftDb
-      }&size=${state.collectionSize || state.totalCardsToShow}${localesQueryParam}${debug}${flatFile}`,
+      endpoint: `https://${state.endpoint
```

Comment:
Is this formatting expected?

Reply:
@sanrai 
Not expected, but it makes it more legible (imo).